client/src/components/application/Transactions.vue

```diff
@@ -325,49 +341,38 @@
     background-color: var(--d-grey);
   }
   #transaction-list-wrapper {
-    flex: 1;
-    height: 100vh;
     overflow: auto;
-    padding: 0;
     position: relative;
+    background-color: white;
   }
   #transaction-list {
-    overflow: auto;
-    padding: 0;
     position: absolute;
   }
   .transaction-item {
     padding: 20px 40px;
   }
-  .transaction-item:nth-child(even) {
+  .transaction-item:nth-child(odd) {
     background-color: var(--xl-grey);
   }
-  .transaction-names {
+  .transaction-name {
     border-top: thin solid var(--grey);
   }
-  .transaction-names:first-of-type {
+  .transaction-name:first-of-type {
     margin-top: 16px;
   }
-  .transaction-names:last-of-type {
+  .transaction-name:last-of-type {
     border-bottom: thin solid var(--grey);
   }
   .border-x {
     border-left: thin solid var(--l-grey);
     border-right: thin solid var(--l-grey);
   }
-  .decision-text {
-    font-size: 12px;
-    margin-left: 24px;
-    padding: 0;
-    margin: 0;
-    position: relative;
-    display: block;
-  }
-  .name-option {
-    font-size: 18px;
-    margin-top: 4px;
+
+  /* Helper classes missing from Vuetify: */
```

Comment:
As you've probably noticed, this old version of Vuetify has some helper classes but not all of the ones we're used to. This can be deleted if/when Vuetify is updated in this project.